### PR TITLE
fix PARTUUID after dd copy of raw device && skip ZRAM on RPi4 8GB

### PR DESCRIPTION
**Pull request by mstormi**
To avoid that the external partition is mounted on boot because it has the same PARTUUID as the internal SD.

Apologies for the zram issue I mixed in (I forgot to branch again).
Everybody with a 8GB RPi seems to have this problem for the time being so I thought I'd drop ZRAM on this HW until a fix or workaround is found.

Signed-off-by: Markus Storm <markus.storm@gmx.net>

### CODE REVIEW

File: functions/backup.bash [hunk 1]
```diff
@@ -377,7 +377,8 @@ mirror_SD() {
   fi
   if [[ "$1" == "raw" ]]; then
     echo "Creating a raw partition copy, be prepared this may take long such as 20-30 minutes for a 16 GB SD card"
-    if ! cond_redirect dd if="${src}" bs=1M of="${dest}"; then echo "FAILED (raw device copy)"; return 1; fi
+    if ! cond_redirect dd if="${src}" bs=1M of="${dest}"; then echo "FAILED (raw device copy)"; dirty="yes"; fi
+    if ! cond_redirect echo "y" | set-partuuid "${dest}" random; then echo "FAILED (set random PARTUUID)"; dirty="yes"; fi
```
Comment (ecdye) [inline functions/backup.bash:381]: don't use `echo "y"` rather use the `yes` command

File: includes/set-partuuid [hunk 2]
```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+# shamelessly copied from https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=196778
```
Comment (ecdye) [inline includes/set-partuuid:3]: Are we allowed to do this from a licensing standpoint? Just checking.

**Review by ecdye — CHANGES_REQUESTED** [review]

File: includes/set-partuuid (hunk 2 above)
Comment (mstormi) [reply includes/set-partuuid:3]: Dunno. It's a post offering the code for download, with no license to apply (at least I have not seen any).
So crediting the origin is _more_ than the author asked for in terms of licensing, isn't it ?

File: functions/backup.bash (hunk 1 above)
Comment (mstormi) [reply functions/backup.bash:381]: didn't know it. at your own risk ...

File: includes/set-partuuid (hunk 2 above)
Comment (ecdye) [reply includes/set-partuuid:3]: Sure, it is now. I don't think it matters too much I was more of just curious.

File: functions/backup.bash (hunk 1 above)
Comment (ecdye) [reply functions/backup.bash]: Yep

**Review by ecdye — APPROVED** [review]
LGTM

**Pull request merged**